### PR TITLE
Add russian translation

### PR DESCRIPTION
**Pull request by JekRock**
Add Russian translation for the app

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE resources [
+    <!ENTITY appname "Slideshow">
+    <!ENTITY author "Michael Standen">
+    ]>
+<resources>
+    <string name="app_name"></string>
```
Comment (theturrible) [inline app/src/main/res/values-ru/strings.xml]: missing &appname;

Comment (JekRock) [inline app/src/main/res/values-ru/strings.xml]: I missed it. Can you tell how to fix it without creating a new pull request?

Comment (theturrible) [inline app/src/main/res/values-ru/strings.xml]: @JekRock I'm not sure, but this guy seems like he does [stackoverflow](http://stackoverflow.com/a/7947337)

Comment (JekRock) [inline app/src/main/res/values-ru/strings.xml]: @theturrible now it looks nice)

**Comment by ScreamingHawk** [comment]
Are you able to rearrange the blocks so that they are consistent with the other strings.xml files? 

The groupings are logical within the app and changing them to (almost) alphabetical makes it difficult to see which strings are related. Also, it's hard to look from one to the other to check everything is there when the content has been shuffled around.

**Comment by JekRock** [comment]
Yes, I can do it. I just used IDEA automatic tool to work with translate, it looks to be more comfortable to use it.

File: app/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -1,49 +1,72 @@
...
-    <string name="pref_description_image_details">Отобразить информацию изображения во время слайд-шоу.</string>
-    <string name="pref_description_random_order">Отобразить изображения во время слайд-шоу в случайном порядке.</string>
-    <string name="pref_description_reverse_order">Отобразить изображения во время слайд-шоу в обратном порядке.</string>
+
+	<!-- Credits -->
+    <string name="title_activity_credits">О приложении</string>
+    <string name="credits_creator">Создано <a href="https://michael.standen.link">&author;</a></string>
+    <string name="credits_content1">Slideshow это простое приложение для просмотра изображений на Вашем телефоне</string>
+    <string name="credits_content2">Без каких-либо наворотов, это приложение - самое простое приложение для показа слайдов в магазине.</string>
+    <string name="credits_content3">Вы можете посмотреть исходный код приложения, сообщить о проблеме или предложить новые функции на <a href="https://github.com/Screaminghawk/android-slideshow">Github</a>. \t</string>
```
Comment (ScreamingHawk) [inline app/src/main/res/values-ru/strings.xml]: Can you please remove the \t at the end of this line? I assume it was added by the translation tool you used, or was it intentional?

File: app/src/main/res/values-ru/strings.xml [hunk 3]
```diff
@@ -1,49 +1,72 @@
...
+    <string name="credits_content4">Это приложение с открытым исходным кодом и оно полностью бесплатно; лицензировано в соответствии с <a href="https://tldrlegal.com/license/mit-license">MIT license</a>. \t</string>
```
Comment (ScreamingHawk) [inline app/src/main/res/values-ru/strings.xml]: Remove \t from this line too please.

Comment (JekRock) [inline app/src/main/res/values-ru/strings.xml]: I have no idea where that tabs came from 😃
I will fix it.

**Comment by JekRock** [comment]
Okay, now it looks better)

**Comment by ScreamingHawk** [comment]
Thanks a lot for this! 
Sorry for all the back and forth... I know some of it can be a bit tedious. 
It might not have been hard for you but there is no way I could have done this without help.

**Comment by JekRock** [comment]
Glad to help you!
Thank you for reviewing code)

**Pull request merged**